tests/e2e/service-worker.js

```diff
@@ -48,6 +48,13 @@ const unregisterServiceWorker = () => browser.executeAsyncScript(async () => {
   callback();
 });
 
+const wipeAllCaches = () => browser.executeAsyncScript(async () => {
+  const callback = arguments[arguments.length - 1];
+  const cacheNames = await caches.keys();
```

Comment:
I can't think of a time we would want to unregister the service worker but not wipe the cache. So maybe merge this function into the `unregisterServiceWorker` function?